non-msrv-tests/tests/ui/item_option_syntax.stderr

```diff
@@ -1,41 +1,49 @@
-error: expected parentheses: #[derive_where(...)]
- --> tests/ui/item_option_syntax.rs:7:16
+error: key-value macro attributes are not supported
+ --> tests/ui/item_option_syntax.rs:4:1
   |
-7 | #[derive_where = invalid]
-  |                ^
+4 | #[derive_where = invalid]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: unsupported trait, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, Zeroize, ZeroizeOnDrop
+ --> tests/ui/item_option_syntax.rs:4:18
+  |
+4 | #[derive_where = invalid]
+  |                  ^^^^^^^
+
+error: key-value macro attributes are not supported
+ --> tests/ui/item_option_syntax.rs:7:1
+  |
+7 | #[derive_where = "invalid"]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This extensive error leads to the test failing either for zeroize for no-zeroize

Reply:
See https://github.com/ModProg/derive-where/pull/33/files#r780830288.